content/messaging/agent-chat-api/index.mdx

```diff
@@ -1589,6 +1589,8 @@ It's possible to write to a chat without joining it. The user sending an event w
 
 The method updates the requester's `events_seen_up_to` as if they've seen all chat events.
 
+To set correct bot name use `X-Author-Id` header.
```

Comment:
```suggestion
To call the API as a bot,  use the `X-Author-Id` header set to the bot's name.
```

Reply:
What do you think about such wording? IMO, it puts emphasis on calling the API as a bot (which is the purpose of the header) rather than just naming the bot (as in https://developers.livechat.com/docs/management/configuration-api/#update-bot where you can set the bot's name) .

Reply:
It's better than mine - I was just focusing on my use case (bots name), this is showing broader contex. Before I commit this change I will check if header needs to be set to name or bots id

Reply:
You can actually search & replace this text in all files